vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -392,7 +393,13 @@ class RoomDetailFragment @Inject constructor(
                 return@onEach
             }
             when (mode) {
-                is SendMode.REGULAR -> renderRegularMode(mode.text)
+                is SendMode.REGULAR -> {
+                    if (mode.messageType == MessageType.MSGTYPE_AUDIO) {
```

Comment:
This is where the code is not typed enough IMO, MSGTYPE_AUDIO is not necessarily a voice message. But for draft this is the only possible case I think...

Reply:
I think this is a sign to move away from the `MSGTYPE` and have a separate `Voice` send mode